data/block/block_test.go

```diff
@@ -319,6 +322,49 @@ func TestHeader_GetMiniBlockHeadersWithDstShouldWork(t *testing.T) {
 	assert.Equal(t, uint32(0), hashesWithDest2[string(hash2S0R2)])
 }
 
+func TestHeader_GetOrderedCrossMiniblocksWithDstShouldWork(t *testing.T) {
+	t.Parallel()
+
+	hashSh0ToSh0 := []byte("hash_0_0")
+	hashSh0ToSh1 := []byte("hash_0_1")
+	hash1Sh0ToSh2 := []byte("hash1_0_2")
+	hash2Sh0ToSh2 := []byte("hash2_0_2")
+
+	hdr := &block.Header{
+		Round: 10,
```

Comment:
Setting the round here is useless

Reply:
In shard header implementation it is. Added just to check on assert the same pattern as I did on meta block unit test

Reply:
Actually I will keep this round just to check that it will be set ok in the data.MiniBlockInfo struct from which the round will be used for prints if something failed in the mini block execution, to easily identify the header nonce which contains the mini block which failed